sardana_ni660x/ctrl/Ni660XCTCtrl.py

```diff
@@ -211,13 +211,14 @@ def DeleteDevice(self, axis):
             self.channels[axis].removeListener(self.counterEventReceived)
             self.attributes.pop(axis)
             self.ch_configured.pop(axis)
-        self.channels.pop(axis)
+        ch = self.channels.pop(axis)
```

Comment:
I think this assignment to a variable is not necessary. We don't use it afterwards, right?

Reply:
Yes, fixed now. Thanks!